modules/openxr/scene/openxr_hand.cpp

```diff
@@ -36,6 +36,15 @@
 #include "scene/3d/skeleton_3d.h"
 #include "servers/xr_server.h"
 
+// Bone rotation adjustment for OpenXR rigs. This is an identity quaternion
+// because the incoming quaternions are already in OpenXR format.
+static const Quaternion openxr_bone_adjustment = Quaternion();
```

Comment:
Nitpicking, I would make this an array of size `SKELETON_RIG_MAX`, if we in the future add other options to the enum it's easier to add.

Reply:
Ah yes, good catch. I turned this into a table and moved inside the `_update_skeleton` function to limit the scope.